src/50_Samples_%26_Templates/Star_Rating.html

```diff
@@ -0,0 +1,141 @@
+<!--
+  #### Introduction
+
+  This star rating widget is implemented using only CSS, given AMP's restriction on custom JavaScript. It still has the key features of star rating components:
+
+   * touch, mouse and keyboard accessibility
+   * stars change color when the user mouses over them
+   * once a selection is made, it "sticks"
+   * clean scalable vector icons
+   * screen reader friendly
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/samples_templates/star_rating/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- #### Setup
```

Comment:
Thanks for pointing that out. I must confess that the rules by which text is shown or hidden, elude me. The heading plus text after it, are an example. Will push a fix for that too.